sqlalchemy_history/manager.py

```diff
@@ -130,8 +134,8 @@ def reset(self):
         """
         self.tables = {}
         self.pending_classes = []
-        self.association_tables = set()
-        self.association_version_tables = set()
+        self._association_parent_tables = set()
+        self._association_version_tables = set()
```

Comment:
```
        self.association_parent_tables = set()
        self.association_version_tables = set()
```

First, let not use a vague term like `association_tables` - the more accurate name is `association_parent_tables`
Remove the property `association_tables`